### ISSUE TITLE
additional documentation

### ISSUE THREAD

**Issue by paskuale75** [survos/SurvosApiGridBundle#2]
Is it possible to have a clearer example of how to use this bundle ? Thanks

**Comment by tacman** [comment]
Indeed, it's finally about to be released, now that it works with Symfony's AssetMapper.
